### PR TITLE
Update performing-searches.md

### PR DESCRIPTION
**Pull request by RakeshJesadiya**
Added Example of `in` and `nin` operator.

## Purpose of this pull request

This pull request (PR) demonstrate the example of `in` and `nin` operator for the search field query.

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.3/rest/performing-searches.html
https://devdocs.magento.com/guides/v2.4/rest/performing-searches.html

### CODE REVIEW

File: src/guides/v2.3/rest/performing-searches.md [hunk 1]
```diff
@@ -93,6 +93,29 @@ searchCriteria[filter_groups][0][filters][0][value]=2016-07-01 00:00:00&
 searchCriteria[filter_groups][0][filters][0][condition_type]=gt
 ```
 
+## Simple search using a in conditions type
```
Comment (BarnyShergold) [inline src/guides/v2.3/rest/performing-searches.md:96]:
```suggestion
## Simple search using an `in` conditions type
```

File: src/guides/v2.3/rest/performing-searches.md [hunk 2]
```diff
@@ -93,6 +93,29 @@ searchCriteria[filter_groups][0][filters][0][value]=2016-07-01 00:00:00&
...
+
+The following search finds all the products that are provided in the value field. Using `in` conditions type, you have to pass comma separated values to get in to the response.
```
Comment (BarnyShergold) [inline src/guides/v2.3/rest/performing-searches.md:98]:
```suggestion
The following search finds all the products that are provided in the value field. To use the `in` conditions type, you have to pass comma separated values to get in to the response.
```

Comment (BarnyShergold) [inline src/guides/v2.3/rest/performing-searches.md:98]: I'm not sure for "to get in to the response" if the in refers to the operator or just the word in? Can you reword "you have to pass comma separated values to get in to the response." - as the second half does not make clear sense in English grammar.

File: src/guides/v2.3/rest/performing-searches.md [hunk 3]
```diff
@@ -93,6 +93,29 @@ searchCriteria[filter_groups][0][filters][0][value]=2016-07-01 00:00:00&
...
+
+```http
+GET <host>/rest/<store_code>/V1/products?
+searchCriteria[filter_groups][0][filters][0][field]=entity_id&
+searchCriteria[filter_groups][0][filters][0][value]=1,2,3,4,5&
+searchCriteria[filter_groups][0][filters][0][condition_type]=in
+```
+The query returns 5 items.
+
+## Simple search using a nin conditions type
```
Comment (BarnyShergold) [inline src/guides/v2.3/rest/performing-searches.md:108]:
```suggestion
## Simple search using a `nin` conditions type
```

File: src/guides/v2.3/rest/performing-searches.md [hunk 4]
```diff
@@ -93,6 +93,29 @@ searchCriteria[filter_groups][0][filters][0][value]=2016-07-01 00:00:00&
...
+
+The following search finds all the products that are not in the value field. Using `nin` conditions type, you have to pass comma separated value to get in the response.
```
Comment (BarnyShergold) [inline src/guides/v2.3/rest/performing-searches.md:110]:
```suggestion
The following search finds all the products that are not in the value field. To use the `nin` conditions type, you have to pass comma separated value to get in the response.
```

Comment (BarnyShergold) [inline src/guides/v2.3/rest/performing-searches.md:110]: Second half of sentence has same grammar issue as above - please can this be reworded?

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Just a few grammar and clarity suggestions - nice addition to the page

**Comment by RakeshJesadiya** [comment]
@BarnyShergold I have updated changes. Thank you.

**Comment by keharper** [comment]
@RakeshJesadiya The topic provides a variety of examples, and I don't think we need more. I accepted the `in` example, but deleted the `nin` example.

**Review by keharper — APPROVED** [review]

**Comment by keharper** [comment]
running tests

**Pull request merged**